src/login/Login.test.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import Login, { validateUser } from './Login';
+
+describe('login page', () => {
+  it('Login page loads properly', () => {
+    render(<Login />);
+    expect(screen.getByRole('heading')).toBeInTheDocument();
+    expect(screen.getByLabelText(/username/)).toBeInTheDocument();
+    expect(screen.getByLabelText(/password/)).toBeInTheDocument();
+    expect(screen.getByRole('button')).toBeInTheDocument();
+  });
+
+  it('Initial state for elements', () => {
+    render(<Login />);
+    expect(screen.getByLabelText(/username/)).toHaveValue('');
+    expect(screen.getByLabelText(/password/)).toHaveValue('');
+    expect(screen.getByRole('button')).toBeDisabled();
+  });
+
+  it.each([
+    ['admin', 'admin', true],
+    ['admin', '', false],
+  ])('.add(%i, %i)', (a, b, expected) => {
```

Comment:
Change the name of the test from `.add(...)` to something more descriptive. Also change the variable names form a, b to username and password